.circleci/config.yml

```diff
@@ -201,6 +201,47 @@ jobs:
                 name: Notify Slack on failure
                 when: on_fail
         working_directory: ~/kubernetes-monitor
+    deploy_sysdig_integration_cluster:
+        docker:
+            - auth:
+                password: $DOCKERHUB_PASSWORD
+                username: $DOCKERHUB_USER
+              image: cimg/base:stable
+        steps:
+            - checkout
+            - run:
+                command: |
+                    LATEST_KUBECTL_VERSION=$(curl -L -s https://dl.k8s.io/release/stable.txt)
+                    curl -LO "https://dl.k8s.io/release/${LATEST_KUBECTL_VERSION}/bin/linux/amd64/kubectl"
+                    curl -LO "https://dl.k8s.io/${LATEST_KUBECTL_VERSION}/bin/linux/amd64/kubectl.sha256"
+                    echo "$(<kubectl.sha256) kubectl" | sha256sum --check
+                    sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+                    # Ensure the kubectl command is runnable
+                    kubectl version --client
+                    # Prepare kubeconfig to point to the cluster
+                    mkdir ~/.kube || true
+                    printf "%s" "${SYSDIG_KUBECONFIG}" | base64 -d > ~/.kube/config
+                name: Install and prepare kubectl
```

Comment:
nit: can the `name` be above the command for easier reading?

Reply:
This is the auto-generated config!

Reply:
yeah realised this after 😆 I forget about the complicated k-m circleci stuff